.github/workflows/deploy.yml

```diff
@@ -0,0 +1,62 @@
+name: Deploy Websites
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build and Deploy websites
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: false
+
+      - name: Fetch Submodules
+        run: |
+          pushd themes/docsy;
+          git submodule update -f --init --recursive --depth 1;
+          popd;
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: '0.75.1'
+          extended: true
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
```

Comment:
did you mean to have a trailing dash here?

Reply:
Honestly...nope :-)

Reply:
Actually...maybe...that is correct

Reply:
ahh ok ... it's a prefix .... so yes trailing dash is fine

https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows#input-parameters-for-the-cache-action